aten/src/ATen/native/README.md

```diff
@@ -73,11 +74,15 @@ signature.
 - `std::array<bool,N>` (where N is `1-4`).  NB: you MUST NOT put a space after the comma, otherwise
   this argument will not parse correctly.  (If you decide to fix this, make sure you fix the
   argument parser both in ATen and in PyTorch.)
+- `*` is a special sentinel argument, which doesn't translate into a real
```

Comment:
can you not used the word `real` -- that has another meaning an a similar context :).  Maybe something like "an actual", "a realized" or similar.